modules/ssh.go

```diff
@@ -22,10 +34,14 @@ type SSHScanner struct {
 
 func init() {
 	var sshModule SSHModule
-	_, err := zgrab2.AddCommand("ssh", "SSH Banner Grab", "Grab a banner over SSH", 22, &sshModule)
+	cmd, err := zgrab2.AddCommand("ssh", "SSH Banner Grab", "Grab a banner over SSH", 22, &sshModule)
 	if err != nil {
 		log.Fatal(err)
 	}
+	s := ssh.MakeSSHConfig() //dummy variable to get default for host key, kex algorithm, ciphers
+	cmd.FindOptionByLongName("host-key-algorithms").Default = []string{strings.Join(s.HostKeyAlgorithms, ",")}
+	cmd.FindOptionByLongName("kex-algorithms").Default = []string{strings.Join(s.KeyExchanges, ",")}
+	cmd.FindOptionByLongName("ciphers").Default = []string{strings.Join(s.Ciphers, ",")}
```

Comment:
Why are these set here instead of in the struct tag? Is it just because they're long?

Reply:
Yeah, setting them in the struct tag would just be a long string that I didn't think should be hardcoded. It still shows up in the help docs though. 

I think the answer is to put these long outputs in the Usage part of the help docs, not shown as defaults inline with the flag. And also to set these defaults in the ssh config if alternates are not specified. Does that sound right?

Reply:
Can you paste what the output of `--help` looks like?

Reply:
![zgrab2](https://user-images.githubusercontent.com/6451295/32070733-4785b092-ba5b-11e7-9151-33673f438948.PNG)